src/xunit.performance.api/Model/AssemblyModel.cs

```diff
@@ -175,6 +175,143 @@ private static IEnumerable<(string testCaseName, string metric, IEnumerable<doub
     }
 
     [Serializable]
+    [XmlRoot("BenchmarkScenario")]
+    public sealed class BenchmarkScenario
+    {
+        [XmlAttribute("Name")]
+        public string Name { get; set; }
+
+        [XmlAttribute("Namespace")]
+        public string Namespace { get; set; }
+        
+        [XmlArray("Assemblies")]
+        public List<NetcoreAssembly> Assemblies { get; set; }
+
+        public BenchmarkScenario()
+        {
+            Assemblies = new List<NetcoreAssembly>();
+            Namespace = "";
+        }
+
+        public BenchmarkScenario(string name)
+        {
+            Name = name;
+            Namespace = "";
+            Assemblies = new List<NetcoreAssembly>();
+        }
+
+        public void Serialize(string xmlFileName)
+        {
+            var namespaces = new XmlSerializerNamespaces();
+            namespaces.Add("", "");
+            using (var stream = File.Create(xmlFileName))
+            {
+                using (var sw = new StreamWriter(stream))
+                {
+                    new XmlSerializer(typeof(BenchmarkScenario))
+                        .Serialize(sw, this, namespaces);
+                }
+            }
+        }
+
+        internal DataTable GetStatistics()
+        {
+            var dt = new DataTable();
+            var col0_testName = dt.AddColumn("Test Name");
```

Comment:
Are there constants for these column names?  I think these are duplicated from the xunit runner code, so we should probably have constants for these to avoid breaks when a name is changed.